src/core/settings/settings.cpp

```diff
@@ -118,8 +139,31 @@ QByteArray Settings::json()
     QJsonObject dataStart {
         {"url", m_startConfig->startUrl().toString()},
     };
+    QString searchEngineString;
```

Comment:
it would be better if search engines could be loaded from a data file, this would allow users to install another engine later on.